mpmp/src/model/Player.java

```diff
@@ -101,6 +104,37 @@ public void ragequit() {
 	}
 
 	/**
+	 * Add a unjail card to the player.
+	 */
+	public void addUnjailCard() {
+		unjails++;
+	}
+
+	/**
+	 * Give one unjail card to player p. Usually against monetary compensation.
+	 */
+	public void giveUnjailCard(Player p) {
+		if(unjails <= 0)
+			return;
+
+		unjails--;
+		p.unjails++;
+	}
+
+	/**
+	 * Leave the prison by using a unjail card.
+	 */
+	public void useUnjailCard() {
```

Comment:
You could make both of that methods boolean, so they return `false` if it fails and maybe display an error.